lib/base/Version.ts

```diff
@@ -229,10 +229,31 @@ export default class Version {
     return operationPromise;
   }
 
+  /**
+   * For each record instance, executes a provided callback function with that
+   * instance
+   *
+   * @param {object} [params] Parameters
+   * @param {number} [params.limit] The maximum number of record instances to
+   *  fetch
+   * @param {number} [params.pageSize] The maximum number of records to return
+   *  with every request
+   * @param {function} params.callback Callback function to call with each
+   *  record instance
+   * @param {function} [params.done] Optional done function to call when all
+   *  records are processed, the limit is reached, or an error occurs.
+   *  Receives an error argument if an error occurs.
+   * @param {function} [callback] Callback function to call with each record.
+   *  Receives a done function argument that will short-circuit the for-each
+   *  loop that may accept an error argument.
+   * @returns {Promise<void>} Returns a promise that resolves when all records
```

Comment:
Raghav made changes to docstring to follow [tsdoc](https://github.com/twilio/twilio-node/blob/4.0.0-rc/lib/jwt/AccessToken.ts#L248) recently. would be nice to adhere to it

Reply:
Done